carve.lic

```diff
@@ -12,12 +12,22 @@ class Carve
     settings = get_settings
     @bag = settings.crafting_container
     @belt = settings.engineering_belt
+    @use_bone = settings.use_bone
```

Comment:
Given how many settings we have, I'm not sure `use_bone` is descriptive/specific enough. What about `use_bone_for_carving`?